pygmt/helpers/utils.py

```diff
@@ -205,7 +205,55 @@ def data_kind(data=None, x=None, y=None, z=None, required_z=False, required_data
     return kind
 
 
-def non_ascii_to_octal(argstr: str) -> str:
+def check_encoding(argstr: str) -> str:
+    """
+    Check the charset encoding of a string.
+
+    All characters in the string must be in a single charset encoding, otherwise the
+    default ISOLatin1+ encoding is returned. Characters in the Symbol and ZapfDingbats
+    fonts are also checked because they're independent on the setting of charset.
```

Comment:
```suggestion
    All characters in the string must be in the same charset encoding, otherwise the
    default ISOLatin1+ encoding is returned. Characters in the Symbol and ZapfDingbats
    fonts are also checked because they're independent on the setting of the charset.
```